### PR TITLE
bpo-38787: update docs of structures.rst and code style of _testmultiphase.c

### PR DESCRIPTION
**Pull request by shihai1991**
https://bugs.python.org/issue38787

### CODE REVIEW

File: Modules/_testmultiphase.c [hunk 1]
```diff
@@ -820,16 +820,12 @@ static PyModuleDef_Slot meth_state_access_slots[] = {
 };
 
 static PyModuleDef def_meth_state_access = {
-    PyModuleDef_HEAD_INIT,                      /* m_base */
```
Comment (corona10) [inline Modules/_testmultiphase.c:823]: We don't usually accept the style changing PR.

**Review by corona10 — CHANGES_REQUESTED** [review]
I left a comment

File: Modules/_testmultiphase.c (hunk 1 above)
Comment (vstinner) [reply Modules/_testmultiphase.c:823]: I asked for this PR on purpose :-) Read comments starting at: https://github.com/python/cpython/pull/19936#pullrequestreview-405978698

**Review by vstinner — APPROVED** [review]
LGTM.

File: Modules/_testmultiphase.c (hunk 1 above)
Comment (corona10) [reply Modules/_testmultiphase.c:823]: I see :)

**Review by corona10 — APPROVED** [review]
lgtm

**Comment by vstinner** [comment]
Thanks @shihai1991, merged.

**Comment by shihai1991** [comment]
Thanks, folks.

**Pull request merged**